src/resources/views/crud/form_content.blade.php

```diff
@@ -100,7 +100,7 @@ function initializeFieldsWithJavascript(container) {
       // Place the focus on the first element in the form
       @if( $crud->getAutoFocusOnFirstField() )
         @php
-          $focusField = Arr::first($fields, function($field) {
+          $focusField = Arr::first($crud->fields(), function($field) {
```

Comment:
Needed if we remove passing the `$fields` in the view as a parameter. It's not needed there because we already have the fields inside `$crud`.